DiztinGUIsh/static/Data.cs

```diff
@@ -210,13 +223,22 @@ public static void SetMXFlags(int i, int mx)
             table[i].XFlag = ((mx & 0x10) != 0);
         }
 
-        public static string GetLabel(int i)
+        public static string GetLabelName(int i)
+        {
+            if (alias.TryGetValue(i, out AliasInfo val)) 
+                return val?.name ?? "";
+
+            return "";
+        }
+        public static string GetLabelComment(int i)
         {
-            if (alias.TryGetValue(i, out string val)) return val;
+            if (alias.TryGetValue(i, out AliasInfo val)) 
+                return val?.comment ?? "";
```

Comment:
I was picking up a few edge cases where if labels were deleted, they would show up as null.  We handle this case now but, should probably get to the root of why null aliases are getting into the main Dictionary